src/eduid_userdb/userdb.py

```diff
@@ -288,3 +288,19 @@ def update_user(self, obj_id, attributes):
                         '$set': attributes,
                     }
                 )
+
+    def get_identity_proofing(self, user):
+        """
```

Comment:
I didn't look into this method, I just moved it over from DasboardUser, because after signing up it needs to be called after the user is in the central db but before it is in the dashboard  db.